catroid/build.gradle

```diff
@@ -189,6 +190,9 @@ dependencies {
     androidTestCompile ('com.android.support.test.espresso:espresso-contrib:2.2.2') {
         exclude group: 'com.android.support'
     }
+    androidTestCompile('com.android.support.test.espresso:espresso-intents:2.2') {
```

Comment:
Oh. Though I did select it completely and then hit (Ctrl + Alt + L) and still missed.